Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile used only for the API tests.
 
-FROM golang:1.16
+FROM golang:1.18.0
```

Comment:
I think better to use `1.18` without specifying patch version.

Reply:
Agree